doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -2,243 +2,33 @@
-
-### Changelog
-
-Update the CHANGELOG.md file which exists in `Azure.<group>.<service>/CHANGELOG.md`. For general information about the changelog, see the [Changelog Guidelines](https://azure.github.io/azure-sdk/policies_releases.html#change-logs).
-
-### Add Convenience APIs
-
-Adding convenience APIs is not required for Azure SDK data plane generated libraries, but doing so can provide customers with a better experience when they develop code using your library.  You should consider adding convenience APIs to the generated client to make it easier to use for the most common customer scenarios, or based on customer feedback.  Any convenience APIs you add should be approved with the Azure SDK architecture board.
-
-You can add convienice APIs by adding a customization layer on top of the generated code.  Please see the [autorest.csharp README](https://github.com/Azure/autorest.csharp#setup) for the details of adding the customization layer.  This is the preferred method for adding convenience APIs to your generated client.
-
-
-If other modifications are needed to the generated API, you can consider making them directly to the Open API specification, which will have the benefit of making the changes to the library in all languages you generate the library in.  As a last resort, you can add modifications with swagger transforms in the `autorest.md` file.  Details for various transforms can be found in [Customizing the generated code](https://github.com/Azure/autorest.csharp#customizing-the-generated-code).
-
-Once you've made changes to the public API, you will need to run the `eng\scripts\Export-API.ps1` script to update the public API listing. This will generate a file in the library's directory similar to the example found in [sdk\template\Azure.Template\api\Azure.Template.netstandard2.0.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/api/Azure.Template.netstandard2.0.cs).
-
-e.g. Running the script for a project in `sdk\deviceupdate` would look like this:
-
-```powershell
-eng\scripts\Export-API.ps1 deviceupdate
-```
-
-### APIView
-
-Once you've done all above requirements, you will need to upload public API to [APIView Website](https://apiview.dev/) for review.
-
-Here are the steps:
-
-- Create the artifact: Run `dotnet pack` under `sdk\<service>\Azure.<group>.<service>` directory. The artifact will be generated to the directory `artifacts\packages\Debug\Azure.<group>.<service>`
-- Upload the artifact to [APIView Website](https://apiview.dev/) to create APIView of the service.
\ No newline at end of file
+Before the library package can be released, you will need to add several requirements manually, including tests, samples, README, and CHANGELOG. Please refer to [Azure SDK Package Ship Requirements](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/Azure_SDK_Package_Ship_Requirements.md) to add those requirements.
```

Comment:
> add several requirements manually [](http://example.com/codeflow?start=60&length=34)

I am assuming we will create all the necessary folder structures for tests, samples and files like readme and changelog and this is mainly updating those with the right content for the SDK. If so, consider changing this to

Before library package can be released, you will need to add tests and samples. You will also need to update the changelogs and README's as necessary.

Reply:
updated. Thanks @pallavit